Granting wish part of #15497: link the source file loaded by "Load" in coqdoc when this file is referred by qualid (#15513)
coq/coq: doc/sphinx/proof-engine/vernacular-commands.rst

(and default) extension of Coq’s script files is .v.

.. cmd:: Load {? Verbose } {| @string | @qualid }

>    Executes the commands of a file in the current context.

Reviewer: ```suggestion
   Executes the commands of a :n:`.v` file in the current context.
```

Is that right--only .v files can be loaded?  If not, is it really useful to load non-.v files?

Reviewer 2: The contents of the file must always be something that's valid for a .v file.  The ability to load files with different extensions might be useful?

Reviewer 2: In particular, if you can Load .out files, then you can combine `Redirect` with `Load` to dynamically emit Gallina (e.g., via Ltac `idtac`) and then run it.

Reviewer: Gotta be careful about "might be useful".
FWIW, in coqtop, Redirect silently overwrites any existing file.  It doesn't do an append.
In CoqIDE, Redirect creates an empty file and prints the idtac in the Messages panel.

   If :n:`@qualid` is specified, :n:`Load` searches
   in the *loadpath* (see Section :ref:`libraries-and-filesystem`)
   for a file with suffix :n:`.v` whose logical name is :n:`@qualid`.

   If :n:`@string` is specified, it must specify a complete filename.
   `~` and .. abbreviations are
   allowed as well as shell variables. If no extension is specified, Coq

>    will use the default extension ``.v``. If the filename is relative,
>    it is searched in the loadpath, ignoring the logical part.

Reviewer: ```suggestion
   appends the default extension ``.v``. If the filename is relative,
   Coq searches in the loadpath, ignoring the logical part of :n:`@qualid`.
```

"ignoring the logical part" -- You mean (for example) strip off `Coq.` from `Coq.Init.Nat`.  Can only be the first component, maybe can say that more explicitly?

   :n:`Verbose` displays the Coq output for each command and tactic
   in the loaded file, as if the commands and tactics were entered interactively.

>    .. exn:: Can’t find file @qualid on loadpath.

Reviewer: Maybe shouldn't refer to `@qualid` as a file?
